WikipediaUnitTests/Code/WMFNotificationTests.m

```diff
@@ -21,6 +23,9 @@ @implementation WMFNotificationTests
 
 - (void)setUp {
     [super setUp];
+
+    [[NSUserDefaults wmf_userDefaults] wmf_resetToDefaultValues];
```

Comment:
This is more of a tech-debt thing, but it would be better if tests didn't rely on user defaults (or any shared state).